<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ilacbilgi</class>
 <widget class="QDialog" name="ilacbilgi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblBarkod">
       <property name="text">
        <string>&amp;Barkod</string>
       </property>
       <property name="buddy">
        <cstring>leBarkod</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leBarkod">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string>0 000 000;_</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblAdi">
       <property name="text">
        <string>&amp;İlaç Adı</string>
       </property>
       <property name="buddy">
        <cstring>leAdi</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leAdi">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblEtkenMad">
       <property name="text">
        <string>&amp;Etken Madde</string>
       </property>
       <property name="buddy">
        <cstring>leEtkenMad</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leEtkenMad">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblForm">
       <property name="text">
        <string>&amp;Form</string>
       </property>
       <property name="buddy">
        <cstring>leForm</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leForm">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblDozaj">
       <property name="text">
        <string>&amp;Dozaj Bilgisi</string>
       </property>
       <property name="buddy">
        <cstring>leDozaj</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leDozaj">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnDuzenle">
       <property name="text">
        <string>Duzenle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCik">
       <property name="text">
        <string>Çık</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCik</sender>
   <signal>clicked()</signal>
   <receiver>ilacbilgi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
